RELEASE-NOTES.md

```diff
@@ -3,8 +3,30 @@
 ### Version 4.0.0-activity-tracker-unauthorised-beta
 **EUI-4226** Fixed an issue with 401/403 errors ignoring that the user is unauthorised.
 
+### Version 3.1.5-complex-hidden-collection-show
```

Comment:
_Nitpick:_
Do we want to sort out the order of versions in here? We've got 4.1.0 coming after a bunch of earlier ones...

Reply:
And a bunch of older versions deleted...? 🤔

Reply:
Will be cleaned up as part of RELEASE PREPS